tests/test_sock.py

```diff
@@ -9,19 +9,10 @@
 @mock.patch('os.getpid')
 @mock.patch('os.unlink')
 @mock.patch('socket.fromfd')
-def test_unix_socket_close_delete_if_exlock(fromfd, unlink, getpid):
+def test_unix_socket_close_keep(fromfd, unlink, getpid):
```

Comment:
Seems like this should read `test_unix_socket_close_unlink` since this test case checks that unlink _is_ called.